File: python-pr/52/python/code-quality-naming/data_formatter_49.py
# {fact rule=code-quality-naming@v1.0 defects=0}
def transform_customer_data_format(raw_customer_data):
    import re
    normalized_data = re.sub(r'[^a-zA-Z0-9]', '', raw_customer_data)
    return normalized_data
# {/fact}

File: js-pr/56/javascript/code-quality-naming/data_formatter_49.js
// {fact rule=code-quality-naming@v1.0 defects=0}
function transformCustomerDataFormat(rawCustomerData) {
    let normalizedData = rawCustomerData.replace(/[^a-zA-Z0-9]/g, '');
    return normalizedData;
}
// {/fact}